node/src/template_chain_spec.rs

```diff
@@ -44,6 +44,18 @@ pub fn chain_spec() -> Result<ChainSpec, EnvVarReadError> {
 			initial_authorities: vec![],
 			main_chain_scripts: read_mainchain_scripts_from_env()?,
 		},
+		native_token_management: NativeTokenManagementConfig {
+			main_chain_scripts: sp_native_token_management::MainChainScripts {
```

Comment:
Is it possible to have `read_main_chains_scripts` in `sp_native_token_management`, that doesn't depend on `sc_*`, so it would be used here like `main_chain_scripts: sp_native_token_management::read_main_chains_scripts` ?

This would adhere to the approach when we move as much code to library crates as possible.

Reply:
I added a helper function in the node crate for now. We have one for the other main chain scripts and the env var reading helpers are here as well, so I din't want to move too much stuff around.

Reply:
I'm strongly for having as much code as possible out of node. I don't say to move other. Leave others for a specific PR with refactor, but lets not add to node if it's relatively easy, please.

If we merge it here and make a release, users will have to C-P it. If we move it out then, users will have to follow refactor.